java-components/cli/pom.xml

```diff
@@ -52,6 +52,10 @@
             <groupId>com.google.cloud.tools</groupId>
             <artifactId>jib-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>commons-logging-jboss-logging</artifactId>
+        </dependency>
```

Comment:
Without this, CLI operations like download were failing (even from `main`)

Reply:
Odd, I wonder if it is related to the Quarkus upgrade.